example/tools/lint/BUILD.bazel

```diff
@@ -8,6 +8,7 @@ load("@aspect_bazel_lib//lib:copy_to_directory.bzl", "copy_to_directory")
 load("@aspect_rules_lint//lint:vale_library.bzl", "VALE_STYLES")
 load("@bazel_skylib//rules:native_binary.bzl", "native_binary")
 load("@npm//:eslint/package_json.bzl", eslint_bin = "bin")
+load("@npm//:stylelint/package_json.bzl", stylelint_bin = "bin")
```

Comment:
Makes me happy to see this all fit together with @npm dep linters slotting in easily